ui/react-components/api/recurringAppointmentsApi.js

```diff
@@ -1,5 +1,5 @@
 import axios from "axios";
-import {recurringAppointmentsSaveUrl} from "../constants";
+import {recurringAppointmentByUuidUrl, recurringAppointmentsSaveUrl} from "../constants";
```

Comment:
can we get recurringAppointmetn without UUID? why do we call the constant as ByUuidUrl?

Reply:
Done